pkg/services/ngalert/api/lotex_ruler.go

```diff
@@ -203,10 +204,14 @@ func (r *LotexRuler) validateAndGetPrefix(ctx *models.ReqContext) (string, error
 	subtype := ctx.Query(subtypeQuery)
 	subTypePrefix, ok := subtypeToPrefix[subtype]
 	if !ok {
-		r.log.Debug("unable to determine prometheus datasource subtype, using default prefix", "subtype", subtype)
+		r.log.Debug(
+			"unable to determine prometheus datasource subtype, using default prefix",
```

Comment:
Style nits: we're supposed to start log messages with a capital letter now.

Reply:
Sorry I forgot to fix this before I merged. I've opened https://github.com/grafana/grafana/pull/57922. Do you have a link to where this is mentioned?